pallets/compliance-manager/src/benchmarking.rs

```diff
@@ -21,23 +21,61 @@ use polymesh_common_utilities::{
     benchs::{AccountIdOf, User, UserBuilder},
     TestUtilsFn,
 };
-use polymesh_primitives::{asset::AssetType, TrustedFor, TrustedIssuer};
+use polymesh_primitives::{asset::AssetType, ClaimType, Scope, TrustedFor, TrustedIssuer};
+use sp_std::convert::TryFrom;
 
 const MAX_DEFAULT_TRUSTED_CLAIM_ISSUERS: u32 = 3;
 const MAX_TRUSTED_ISSUER_PER_CONDITION: u32 = 3;
 const MAX_SENDER_CONDITIONS_PER_COMPLIANCE: u32 = 3;
 const MAX_RECEIVER_CONDITIONS_PER_COMPLIANCE: u32 = 3;
 const MAX_COMPLIANCE_REQUIREMENTS: u32 = 2;
 
+const MAX_CONDITIONS: u32 = 10;
+const MAX_CONDITION_TYPE_CLAIMS: u32 = 10;
+const MAX_CONDITION_ISSUERS: u32 = 10;
+const MAX_CONDITION_ISSUER_CLAIM_TYPES: u32 = 10;
+/*
+const MIN_CONDITIONS: u32 = 1;
+const MIN_CONDITION_TYPE_CLAIMS: u32 = 1;
+const MIN_CONDITION_ISSUERS: u32 = 1;
+const MIN_CONDITION_ISSUER_CLAIM_TYPES: u32 = 1;
+*/
+
+const CLAIM_TYPES: &'static [ClaimType] = &[
+    ClaimType::Accredited,
+    ClaimType::Affiliate,
+    ClaimType::BuyLockup,
+    ClaimType::SellLockup,
+    ClaimType::CustomerDueDiligence,
+    ClaimType::KnowYourCustomer,
+    ClaimType::Jurisdiction,
+    ClaimType::Exempted,
+    ClaimType::Blocked,
+    ClaimType::InvestorUniqueness,
+    ClaimType::NoType,
+    ClaimType::InvestorUniquenessV2,
+];
```

Comment:
Let's define this as an associated constant to `ClaimType` instead and close-by so that it's easier to keep it updated.

Also `'static` is redundant on `const` and `static` items as it is inferred and is the only possible lifetime in non-polymorphic cases (i.e. `for<'a> fn(...)`) given we don't have generic constants/statics.

Reply:
It only has use in the benchmarks.  It is fine where it is.

Reply:
Yeah but you'll need to remember to add new variants to it whenever `ClaimType` is extended, no?